src/__tests__/accommodationAction.spec.js

```diff
@@ -0,0 +1,18 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+
+const createMockStore = configureMockStore([thunk]);
+const store = createMockStore({ accommodation: {}});
+
+import {viewAllAccommodations} from '../redux/actions/accommodationAction';
+describe('Accommodations action creator tests', () => {
+    // beforeEach(() => {
+    //     jest.setTimeout(50000);
+    //   });
```

Comment:
@hakoemmy remove the comments if the code is no longer needed

Reply:
@AbdoulNuru Thanks for that. I removed the comments